community/modules/compute/schedmd-slurm-gcp-v6-nodeset/variables.tf

```diff
@@ -414,24 +414,24 @@ variable "additional_networks" {
   description = "Additional network interface details for GCE, if any."
   default     = []
   type = list(object({
-    network            = string
-    subnetwork         = string
-    subnetwork_project = string
-    network_ip         = string
-    nic_type           = string
-    stack_type         = string
-    queue_count        = number
-    access_config = list(object({
+    network            = optional(string)
+    subnetwork         = optional(string)
```

Comment:
nit. I would recommend to make `subnetwork` non-optional since it's an "intended" way to pass "VPC", though name is misleading due to historic reasons (should be `subnetwork_self_link`).

Reply:
done